src/app/charts/core/templates/_helpers.tpl

```diff
@@ -834,3 +834,24 @@
 
   {{- $result | uniq | toYaml -}}
 {{- end -}}
+
+{{- define "__arkcase.env.applicationVars.compute.active" -}}
+{{- $portal := (include "arkcase.portal" . | fromYaml) -}}
+{{- list "arkcase" "arkcase-activemq" "arkcase-oidc" "ldap" "lookups" ((empty $portal) | ternary "" "arkcase-portal") | compact | join "," -}}
+{{- end }}
+
+{{- define "__arkcase.env.applicationVars.compute.profile" -}}
+{{- $portal := (include "arkcase.portal" . | fromYaml) -}}
+{{- $custom := (include "arkcase.customization" .) -}}
+{{- $customBase := printf "%s_base" $custom -}}
+{{- (empty $portal) | ternary (list $customBase $custom "server" "runtime") (list $customBase "foia" $custom "server" "FOIA_server" "runtime") | compact | join "," -}}
+{{- end }}
+
+{{- define "arkcase.env.applicationVars" }}
+- name: ARKCASE_APPLICATION_ACTIVE
+  value: "{{ include "__arkcase.env.applicationVars.compute.active" $ }}"
+- name: ARKCASE_APPLICATION_PROFILE
+  value: "{{ include "__arkcase.env.applicationVars.compute.profile" $ }}"
+- name: ARKCASE_INTERNAL_CONFIG_SERVER_BUCKET
+  value: "arkcase-config"
```

Comment:
This won't be necessary. The config server's internal content store is fully abstracted out.  The configuration code that writes files into there should simply inject a reference to `com.armedia.acm.content.InternalContentStore`, and call it a day.